### PR TITLE
Service accounts: Update documentation with new changes

### PR DESCRIPTION
**Pull request by vtorosyan**
**What this PR does / why we need it**:

Grafana service accounts will be GA released with 9.1, this PR updates the documentation with last remaining changes.

**Which issue(s) this PR fixes**:

Fixes https://github.com/grafana/grafana-enterprise/issues/3501

**Special notes for your reviewer**:

I have copy pasted the API specs for adding roles to users and created a new one for managing roles for service accounts. As the API is actually the same, I thought this will it will be much clear and discoverable, compared to living it with users usages only and updating the text.

**To-do**
- [x] Add information about migrating API Keys to service accounts

### CODE REVIEW

**Comment by vtorosyan** [comment]
@IevaVasiljeva @Jguer you can already review this for initial feedback - as from my side, I just need to add the part for migrating API Keys to service accounts. I just missed the knowledge about how we actually ended up doing it, so would need to catchup about that with you.

**Review by Jguer — APPROVED** [review]

File: docs/sources/administration/team-management/_index.md [hunk 1]
```diff
@@ -86,8 +88,8 @@ You can remove a team member when you no longer want to apply team permissions t
 
 1. Sign in to Grafana as an organization administrator or team administrator.
 1. Hover your cursor over the **Configuration** (gear) icon in the side menu and click **Teams**.
-1. Click a team from which you want to remove a user.
-1. Click the **X** next to the name of the user.
+1. Click a team from which you want to remove a user or a service account.
+1. Click the **X** next to the name of the user or a service account.
```
Comment (IevaVasiljeva) [inline docs/sources/administration/team-management/_index.md:92]: nit:
```suggestion
1. Click the **X** next to the name of the user or service account.
```

**Review by IevaVasiljeva — COMMENTED** [review]
Looks good overall! I've left some suggestions in comments.

One more thing that would be nice to document is service account managed permissions (the ones added in https://github.com/grafana/grafana/pull/51818).

File: docs/sources/administration/roles-and-permissions/access-control/custom-role-actions-scopes.md [hunk 2]
```diff
@@ -101,6 +101,12 @@ The following list contains role-based access control actions.
...
+| `serviceaccounts:read`               | `serviceaccounts:*`                                                                     | Read Grafana service accounts.                                                                                                                                                                   |
+| `serviceaccounts.permissions:write`  | `serviceaccounts:*`                                                                     | Update Grafana service account permissions.                                                                                                                                                      |
+| `serviceaccounts.permissions:read`   | `serviceaccounts:*`                                                                     | Read Grafana service account permissions.                                                                                                                                                        |
```
[the remark this answers is not in the document]
Comment (vtorosyan) [reply docs/sources/administration/roles-and-permissions/access-control/custom-role-actions-scopes.md:109]: +1 for the first thought. I will update to mention that those permissions apply also to service accounts. Regarding the second idea, smells like indeed we want to make it clear. Let's follow up in person about it.

**Comment by vtorosyan** [comment]
Merging this PR - please feel free to continue providing feedback and happy to address if anything needs and can be improved.

**Pull request merged**